lib/Phpfastcache/Drivers/Redis/Driver.php

```diff
@@ -125,13 +125,16 @@ protected function driverConnect(): bool
      */
     protected function driverReadAllKeys(string $pattern = '*'): iterable
     {
-        $i = -1;
-        $keys = $this->instance->scan($i, $pattern === '' ? '*' : $pattern, ExtendedCacheItemPoolInterface::MAX_ALL_KEYS_COUNT);
-        if (is_iterable($keys)) {
-            return $keys;
-        } else {
-            return [];
-        }
+        $i = null;
+        $keys = [];
+        $pattern = $pattern === '' ? '*' : $pattern;
+        do {
+            $keys = array_merge($keys, $this->instance->scan($i, $pattern));
```

Comment:
Do not use` array_merge` in loop due to performance issues.
See this thread:
https://dev.to/klnjmm/never-use-arraymerge-in-a-for-loop-in-php-5go1

Also please canonicalize php core functions:
https://github.com/PHPSocialNetwork/phpfastcache/blob/master/CODING_GUIDELINE.md#php-core-functions